api/campaigns/class-maybe-show-campaign.php

```diff
@@ -118,10 +133,58 @@ function ( $post_data ) use ( $visit ) {
 
 			$response[ $campaign->id ] = $campaign_should_be_shown;
 		}
+
 		$this->response = $response;
 		$this->respond();
 	}
 
+	/**
+	 * Get the highest-priority segment that matches the client.
```

Comment:
Nit: it's a bit fuzzy, cause the "best" priority is the _lowest_ priority integer ;)

Reply:
Ah yes, I had started out calling it the "highest priority" segment everywhere, but switched to calling it "best priority" instead for this exact reason. Updated the last remaining mentions of "highest priority" in 9609176.